pkg/agent/node-resource-topology.go

```diff
@@ -84,9 +84,13 @@ func (a *agent) updateNrtCR(policy string, zones []*policyapi.TopologyZone) erro
 
 	// otherwise update CR if one exists
 	if cr != nil {
-		cr.TopologyPolicies = []string{
-			policy,
+		cr.Attributes = nrtapi.AttributeList{
+			nrtapi.AttributeInfo{
+				Name:  "TopologyPolicy",
+				Value: policy,
```

Comment:
I wonder if we should call this rather `nriPlugin` instead of `TopologyPolicy` since the key can be anything really. Let me know what you think please.

Reply:
thanks for the thoughts @klihub, then let's keep it for now as it is.